index.html

```diff
@@ -242,207 +229,153 @@ <h2 class="primary-text">Multi-Post Stories</h2>
-                  fill="#505F79"
-                />
-              </svg>
-            </li>
-          </ul>
-          <hr />
-        </li>
-      </ul>
+          </li>
+          <li>
+            <a href="https://twitter.com/">
+              <img src="./assets/images/twitter.png" alt="twitter image" />
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="./assets/images/medium.png" alt="Medium image" />
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="btn">Get my resume</button>
+      </div>
+      <div class="about-right">
+        <ul class="drop-options">
+          <li class="drop-options-list">
+            <ul class="hidden-options-title">
+              <li>Languages</li>
+              <li>
+                <svg
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z"
+                    fill="#5E6C84"
+                  />
+                </svg>
+              </li>
+            </ul>
+            <ul class="hidden-options-details">
+              <a href="#">
+                <div class="language-list-item">
+                  <img
+                    class="language-list-item-image"
+                    src="./assets/images/js.png"
+                    alt="Javascript image"
+                  />
+                  <p class="language-list-item-text">Javascript</p>
+                </div>
+              </a>
+              <a href="#">
+                <div class="language-list-item">
+                  <img
+                    class="language-list-item-image"
+                    src="./assets/images/html.png"
+                    alt="Javascript image"
+                  />
+                  <p class="language-list-item-text">HTML</p>
+                </div>
+              </a>
+              <a href="#">
+                <div class="language-list-item">
+                  <img
+                    class="language-list-item-image"
+                    src="./assets/images/css.png"
+                    alt="Javascript image"
+                  />
+                  <p class="language-list-item-text">CSS</p>
+                </div>
+              </a>
+            </ul>
+          </li>
```

Comment:
:red_circle:   Kindly reduce your cards' width and leave some room at the right of the languages list to match the Figma design

Your Design
![tech](https://user-images.githubusercontent.com/26736582/199823114-a56ccd61-0232-487a-a4e8-893be4e26b0e.png)

Figma Design
![tecnologie list](https://user-images.githubusercontent.com/26736582/199822740-b1a7ed5e-3934-40d5-b72e-f467d06a4048.png)